cub/cub/device/dispatch/dispatch_transform.cuh

```diff
@@ -178,15 +190,14 @@ struct dispatch_t<StableAddress,
   // TODO(bgruber): I want to write tests for this but those are highly depending on the architecture we are running
   // on?
   template <typename ActivePolicy>
-  CUB_RUNTIME_FUNCTION _CCCL_VISIBILITY_HIDDEN _CCCL_FORCEINLINE auto configure_ublkcp_kernel() -> cuda_expected<
-    ::cuda::std::
-      tuple<THRUST_NS_QUALIFIER::cuda_cub::detail::triple_chevron, decltype(kernel_source.TransformKernel()), int>>
+  CUB_RUNTIME_FUNCTION _CCCL_VISIBILITY_HIDDEN _CCCL_FORCEINLINE auto configure_ublkcp_kernel(ActivePolicy policy = {})
```

Comment:
critical: as written, this function doesn't work because of:
```cpp
(static auto cached_config = determine_element_counts(); return cached_config;),
```
Dispatch is instantiated for type-erased `indirect_arg` which means that we cache kernel config for the first type and then use it throughout the execution. Test fails when we apply transformation to increasing type sizes, like `int32, int64` because we cache kernel config for small type and then try to apply the same number of items for larger type that doesn't necessarily fit.

Reply:
Ah! That's a really good catch.